app/image/image-details-directive.html

```diff
@@ -52,22 +52,27 @@
       <div class="form-group">
         <label class="col-sm-2 control-label"><span class="glyphicon glyphicon-qrcode"></span> ID</label>
         <div class="col-sm-10">
-          <p class="form-control-static"><a href="image/{{imageDetails.id}}">{{imageDetails.id | limitTo: 12}}</a></p>
+          <p class="form-control-static">
+          <!-- <a href="image/{{imageDetails.id}}">{{imageDetails.id | limitTo: 12}}</a> -->
+            {{imageDetails.id | limitTo: 12}}
+          </p>
         </div>
       </div>
       <div class="form-group">
         <label class="col-sm-2 control-label"><span class="glyphicon glyphicon-arrow-up"></span> Parent's ID</label>
         <div class="col-sm-10">
-          <p class="form-control-static"><a href="image/{{imageDetails.parent}}">{{imageDetails.parent | limitTo: 12}}</a></p>
+          <p class="form-control-static"><!-- <a href="image/{{imageDetails.parent}}">{{imageDetails.parent | limitTo: 12}}</a> -->
+            {{imageDetails.parent | limitTo: 12}}
+          </p>
         </div>
       </div>
-      <div class="form-group">
+      <div class="form-group" ng-hide="true">
         <label class="col-sm-2 control-label"><span class="glyphicon glyphicon-compressed"></span> Size <small>(not including base image sizes)</small></label>
         <div class="col-sm-10">
           <p class="form-control-static">{{imageDetails.Size/1024/1024 | number: 2}} <b>MB</b> {{imageDetails.Size / 1024 | number: 2}} <b>KB</b> {{imageDetails.Size}} <b>B</b></p>
         </div>
       </div>
-      <div class="form-group">
+      <div class="form-group" ng-hide="true">
```

Comment:
I suggest to not leave this div in the code for now, when there's no image size calculated.